admin/views/school/pupils-not-taking-check.ejs

```diff
@@ -0,0 +1,40 @@
+<div class="grid-row">
+    <div class="column-one-half">
+        <%- partial('partials/_breadcrumbs', { breadcrumbs: breadcrumbs }) %>
+    </div>
+    <div class="column-one-half">
+        <%- partial('partials/_name_and_sign_out') %>
+    </div>
+</div>
+
+<div class="grid-row" aria-describedby="heading-xlarge">
+    <div class="column-two-thirds" id="checkWindowsForm">
+        <h1 class="heading-xlarge" id="heading-xlarge" tabindex="0">Pupils not taking the check</h1>
+
+        <p class="lede">Click on the button below to add reasons for pupils who are not taking the check</p>
+
+        <p>
+            <a class="button button-primary" href="#">Add reason for pupil</a>
+        </p>
+
+        <br />
```

Comment:
Why are all these `br`s here?

Reply:
@jon-shipley to add space between the button and the back to top link, so to match the prototype. I don't want a new css class just to add space.